terraform/mesh_lambdas/locals.tf

```diff
@@ -0,0 +1,11 @@
+locals {
+  name = "${var.name_prefix}-mesh"
+
+  mesh_url = {
+    integration = "https://msg.intspineservices.nhs.uk/"
+    production  = "https://mesh-sync.spineservices.nhs.uk"
```

Comment:
production should be https://msg.spineservices.nhs.uk

Reply:
Updated

But maybe https://digital.nhs.uk/developer/api-catalogue/message-exchange-for-social-care-and-health-api#api-description__environments-and-testing needs updating too?